numpy/core/src/common/npy_config.h

```diff
@@ -8,7 +8,7 @@
 #include "numpy/npy_cpu.h"
 #include "numpy/npy_os.h"
 
-/* blacklist */
```

Comment:
If we're going to change this, I think we should expand on this comment. There are various issues around the issue tracker referring to a blacklist of functions. By changing the only word in this comment, it shares no words with any of those former issues!

Reply:
> That is highly inappropriate Chuck, kind of upside down.

It doesn't look that way to me. In any case, what we are seeing here is politics entering into numpy development, and I want to avoid that. I'd also have to live with my cowardice if I didn't speak up, and I want to avoid that also.

Reply:
Didn't mean to bring politics into numpy development, I think that's a bit exaggerate. My only goal was to submit a PR to make the numpy devs aware of these changes that other open source projects are making (didn't explicitly create an issue for this). Besides I think it's perfectly fine to speak out what you think and I hope we can keep doing that, just not sure if a PR would be the best place. Thanks for your input anyway :) @charris